pkg/error/error.go

```diff
@@ -20,6 +22,13 @@ type ValidationErrors struct {
 	Errors   []ValidationError
 }
 
+func (error1 *ValidationError) Equal(error2 ValidationError) bool {
+	return error1.Message.Error() == error2.Message.Error() &&
+		error1.LineNumber == error2.LineNumber &&
+		error1.AdditionalIdenticalErrorCount == error2.AdditionalIdenticalErrorCount
```

Comment:
I don't get the test on the additional identical count

Could you explain?

Reply:
Since `Equal()` is used in the test case, it needs to verify all properties - else once might have a test case succeed that just reports the first error and does not report a correct ending number. This method is not used for the actual consolidation - it would have to be another method `Similar()` (which than would need to ignore the `AdditionalIdenticalErrorCount`

Reply:
Ok it makes sense now